src/components/frame/v5/pages/UserAdvancedPage/partials/UserAdvancedSettings/UserAdvancedSettings.tsx

```diff
@@ -1,42 +1,109 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
+import { FormattedMessage, useIntl, defineMessages } from 'react-intl';
 
 import Navigation from '~v5/common/Navigation';
 import TwoColumns from '~v5/frame/TwoColumns';
 import Spinner from '~v5/shared/Spinner';
 import SettingsRow from '~v5/common/SettingsRow';
 import FeesForm from '../FeesForm';
 import RpcForm from '../RpcForm/RpcForm';
+import Modal from '~v5/shared/Modal';
+import InformationList from '~v5/shared/InformationList/InformationList';
+import { modalInformations } from './consts';
+import Textarea from '~v5/common/Fields/Textarea';
+import Checkbox from '~v5/common/Checkbox/Checkbox';
 
 const displayName = 'v5.pages.UserAdvancedPage.partials.UserAdvancedSettings';
 
-const UserAdvancedSettings: FC = () => (
-  <Spinner loadingText={{ id: 'loading.userAdvancedPage' }}>
-    <TwoColumns aside={<Navigation pageName="profile" />}>
-      <FeesForm />
-      <RpcForm />
-      <div className="border-b border-gray-200">
+const MSG = defineMessages({
```

Comment:
Can you use formatMessage?

Reply:
I had to use <FormattedMessage> because there are <br> in code